lib/portlayer/exec/container.go

```diff
@@ -475,12 +476,20 @@ func (c *Container) stop(ctx context.Context, waitTime *int32) error {
 				}
 				log.Warnf("invalid power state during power off: %s", terr.ExistingState)
 
+			case *types.GenericVmConfigFault:
+				if terr.Reason == notSuspendedMsg {
```

Comment:
Given @hmahmood 's comment, the Key would actually be a better field to check for  `msg.suspend.powerOff.notsuspended` - I don't think these messages are internationalized, but better safe than sorry